students/12th/sonbyungjin/style/login.css

```
@@ -0,0 +1,95 @@
+*{
+  box-sizing: border-box;
+}
+body{
+  display: flex;
```

```suggestion
}

body {
  display: flex;
```
- 중괄호 앞엔 한칸 띄우기
- 새로운 셀렉터 앞에서 한줄 띄우기!